src/Dropdown/Dropdown.jsx

```diff
@@ -251,6 +322,31 @@ const Dropdown = ({
   )
 }
 
+const isEqual = (object1, object2) => {

```

Comment:
I think this function should live in /utils

Reply:
605fde53ac953654bb1190e52d2d242b9fc3a9fd